.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2.1
+orbs:
+  docker: circleci/docker@0.5.19
+  django-tools: pennlabs/django-tools@0.0.2
+  helm-tools: pennlabs/helm-tools@0.1.7
+  react-tools: pennlabs/react-tools@0.0.2
+
+branch-filters: &branch-filters
+  filters:
+    branches:
+      only:
+        - master
+
+workflows:
+  version: 2
+  build-and-deploy:
+    jobs:
+      - react-tools/lint:
+        name: lint-site
+      - docker/publish:
+        name: publish-site
+        requires:
+          - lint-site
+        cache_from: 'pennlabs/website'
```

Comment:
since we're doing a multi stage build, there's no point to doing a cache from since it'll still have to create the build image. this will just take extra time by doing an extra docker pull

Reply:
wouldn't this save time on the `yarn install`? if it doesn't have to do that part of the image build again

Reply:
nah cache from will only pull the nginx image, not the node image that uses to build

Reply:
That’s so wack. Is there not a way to cache the first layer?

Reply:
We could upload the first layer as well, but I don't wanna do that. regardless, it's really not that wack. we can talk about this in person, but it would make no sense for the docker client to always upload and download both the build image and the prod image